test/toucan/db_test.clj

```diff
@@ -169,9 +170,29 @@
 
 ;; TODO - Test simple-delete!
 
-;; TODO - Test simple-insert-many!
+;; Test simple-insert-many!
+(expect
+ '(4 5)
+ (test/with-clean-db
+   (db/simple-insert-many! User [{:first-name "Grass" :last-name #sql/call [:upper "Hopper"]}
+                                 {:first-name "Ko" :last-name "Libri"}])))
 
-;; TODO - Test insert-many!
+(expect
+ '(5)
+ (test/with-clean-db
+   (db/simple-insert-many! Category [{:name "seafood" :parent-category-id 100}])))
+
+;; Test insert-many!
+(expect
+ '(4 5)
+ (test/with-clean-db
+   (db/insert-many! User [{:first-name "Grass" :last-name #sql/call [:upper "Hopper"]}
+                          {:first-name "Ko" :last-name "Libri"}])))
+
+(expect
+ AssertionError ;; triggered by pre-insert hook
```

Comment:
Don't feel the need to change this, but for future reference comments on the same lines as code generally are preceded by a single semicolon, e.g.

```clojure
AssertionError ; triggered by pre-insert hook
```

(We try to strictly adhere to the [Clojure Style Guide](https://github.com/bbatsov/clojure-style-guide). The relevant section is [here](https://github.com/bbatsov/clojure-style-guide#one-semicolon-for-margin-comments).)